linux/keyman-config/keyman_config/sentry_handling.py

```diff
@@ -124,13 +125,24 @@ def _sentry_sdk_initialize(self):
           integrations=[sentry_logging],
           before_send=self._before_send
         )
-        set_user({'id': hash(getpass.getuser())})
+        hash = hashlib.md5()
+        hash.update(getpass.getuser().encode())
+        set_user({'id': hash.hexdigest()})
```

Comment:
I'm not sure about this -- md5 hashes of short strings are pretty easy to reverse with rainbow tables. It's low risk but it is a bit of an overshare.

Reply:
So better use sha256?